_data/engine-cli/docker_config_inspect.yaml

```diff
@@ -6,7 +6,7 @@ long: |-
   By default, this renders all results in a JSON array. If a format is specified,
   the given template will be executed for each result.
 
-  Go's [text/template](http://golang.org/pkg/text/template/) package
+  Go's [text/template](https://golang.org/pkg/text/template/) package
```

Comment:
The YAML files in the `_data/engine-cli` directory are generated from MarkDown source files in the CLI repository. For example, this page is generated from https://github.com/docker/cli/blob/master/docs/reference/commandline/config_inspect.md

So changes made in these YAML files will be overwritten when these files are regenerated and updated. Perhaps you're interested in opening a pull request with these changes in the CLI repository? https://github.com/docker/cli

When doing so, be sure to include a DCO sign-off line (`Signed-Off-By`) line in the commit message (see https://github.com/docker/cli/blob/master/CONTRIBUTING.md#sign-your-work), otherwise CI will fail in that repository.

I'm also happy to make those changes if you want